src/theme/components/banner.ts

```diff
@@ -0,0 +1,164 @@
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+
+import {
+  neutral as baseNeutral,
+  informative as baseInformative,
+  positive as basePositive,
+  negative as baseNegative,
+  warning as baseWarning,
+  recommendation as baseRecommendation,
+} from "./statusBadge";
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(["base", "dismissibleButton"]);
+
+// For now, this semantic mapping is specific to the Banner component.
+const neutral = definePartsStyle({
+  base: {
+    ...baseNeutral,
+    bgColor: "ui.bg.default",
+    svg: {
+      fill: "ui.border.hover",
+      _dark: {
+        fill: "dark.ui.border.hover",
+      },
+    },
+    _dark: {
+      bgColor: "dark.ui.bg.default",
+    },
+  },
+});
+const informative = definePartsStyle({
+  base: {
+    ...baseInformative,
+    svg: {
+      fill: "ui.link.primary",
+      _dark: {
+        fill: "dark.ui.link.primary",
+      },
+    },
+  },
+});
+const positive = definePartsStyle({
+  base: {
+    ...basePositive,
+    svg: {
+      fill: "ui.success.primary",
+      _dark: {
+        fill: "dark.ui.success.primary",
+      },
+    },
+  },
+});
+const negative = definePartsStyle({
+  base: {
+    ...baseNegative,
+    color: "ui.error.primary",
+    a: {
+      color: "ui.error.primary",
+    },
```

Comment:
Note: the links in the "negative" variant are red but in the My Account Figma design they are blue.

Reply:
Is this a problem with the design or something that will be need to be created with an override in My Account ?

Reply:
That is an oversight in the My Account designs.  We need to account for all color posibilities, so the link color should align with the text color for all link states (default, hover, visited).